lib/src/svg/xml_parsers.dart

```diff
@@ -217,7 +217,8 @@ DrawablePaint? parseStroke(
     PaintingStyle.stroke,
     color: rawStroke == ''
         ? (parentStroke?.color ?? colorBlack).withOpacity(opacity)
-        : parseColor(rawStroke)!.withOpacity(opacity),
+        : (parseColor(rawStroke) ?? parentStroke?.color ?? colorBlack)
+            .withOpacity(opacity),
```

Comment:
This still doesn't seem quite right to me. 

I think we want a new property on `SvgPicture` called `currentColor` that will set a default current color. 

Then, we'd want `parseColor` to take an optional `currentColor` parameter, which is initially set to that value, and subsequently set to `parentStroke.color`.

How does that sound to you?

Reply:
I don't know if that's really necessary?  The whole idea is to inherit the color from the parent element if you use `currentColor`, which is exactly what the code currently does if you use `fill=""`.  The change to stroke was specifically to match the behavior of the fill handling, which already did this logic.

Reply:
Yes, but the default for `stroke` is `none`, which I think this will not handle quite right...

Reply:
Or rather, if `none` is explicitly specified.

Reply:
`parseColor` already return `null` for the `none` color.  The code previously failed when `parseColor` returns a null value.  I'm trying to understand the change in behavior here.

Reply:
`parseStroke` also already has an escape clause for this:
```
  if (rawStroke == 'none') {
    return DrawablePaint.empty;
  }
 ```

I guess a test case that fails would help me to understand what's wrong here... do you have a contrived example?

Reply:
Ahh ok. In that case I think this is ok. We could go back and add a property for `currentColor` separately.